.github/workflows/my-ci-pipeline.yml

```diff
@@ -30,4 +30,4 @@ jobs:
    - uses: actions/checkout@v2 
    - run: docker build --target production --tag aexalt/todo-app:latest .
    - run: docker push aexalt/todo-app:latest
-   - run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK }} 
\ No newline at end of file
+   - run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK }} --fail
```

Comment:
After checking your deployment, your website may not be currently working as expected 
![image](https://user-images.githubusercontent.com/92427951/209981782-6d8fd075-1bd5-4b3d-8408-d40aceb1000d.png)

Reply:
apologies not sure why that was, deployed it again and its ok